tools/gnur-runtime/src/main/java/org/renjin/gnur/api/Internal.java

```diff
@@ -1634,9 +1634,9 @@ public static SEXP do_which(SEXP p0, SEXP p1, SEXP p2, SEXP p3) {
     throw new UnimplementedGnuApiMethod("do_which");
   }
 
-  public static SEXP do_withVisible(SEXP p0, SEXP p1, SEXP p2, SEXP p3) {
-    throw new UnimplementedGnuApiMethod("do_withVisible");
-  }
+//  public static SEXP do_withVisible(SEXP p0, SEXP p1, SEXP p2, SEXP p3) {
```

Comment:
Leave this stub as-is. These are intended for C code that call back into the interpreter. Eventually, we can generate these automatically from the data in Primitives.java but for the time being these simply allow C code to compile without error.